packages/apps/job-launcher/server/src/common/exceptions/exception.filter.ts

```diff
@@ -0,0 +1,37 @@
+import { ArgumentsHost, Catch, HttpStatus, Logger } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { DatabaseError } from '../errors/database';
+import { CustomError } from '../errors/custom';
+
+@Catch(CustomError, DatabaseError)
+export class ExceptionFilter implements ExceptionFilter {
```

Comment:
```suggestion
export class CustomExceptionFilter implements ExceptionFilter {
```
otherwise it is implementing itself